tests/phpunit/CRMTraits/Financial/OrderTrait.php

```diff
@@ -45,6 +47,7 @@ public function createRepeatMembershipOrder() {
     $orderID = $this->callAPISuccess('Order', 'create', [
       'total_amount' => '200',
       'financial_type_id' => 'Donation',
+      'source' => 'Online Contribution: form payment',
```

Comment:
Test change is because the template contribution source is now used - the source-is-empty-so-we-need-to-make-something-up is from an older flow IMHO